kilolib.c

```diff
@@ -168,22 +216,101 @@ void kilo_start(void (*setup)(void), void (*loop)(void)) {
                 _delay_us(300);
                 acomp_on();
 
-                set_color(RGB(3,3,3));
-                _delay_ms(10);
-                if (rx_busy) {
+                if(is_in_low_voltage == N_REPETITIONS_LOW_VOLTAGE){ // Low battery
```

Comment:
this change is pretty intrusive to this function. I think we would want to refactor this into several functions that are called from this to keep the main loop somewhat readable.